cves/CVE-2017-12611.yml

```diff
@@ -103,9 +105,13 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
```

Comment:
Is it possible that there were unit tests updated in the commits surrounding the VCC?

Reply:
nope, didn't find any references to unit tests in the code or surrounding commits.